### DISCUSSION TITLE
Bad termination of application

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#26620] [Q&A Modules: Phase field]
Hello, I'm trying to run this file (attached) but I'm getting the error message : 
```

Time Step 0, time = 0

Time Step 1, time = 5e-05, dt = 5e-05
 0 Nonlinear |R| = 4.248216e+02

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 10962 RUNNING AT amval
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Segmentation fault (signal 11)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
```
[bi_material_pf.txt](https://github.com/idaholab/moose/files/14039517/bi_material_pf.txt)

**Comment by ghost** [comment]
I am using two different materials, one is elastic and the other one with phase field damage (ComputeLinearElasticPFFractureStress).

**Reply by GiudGiud** [reply]
I did not see anything obvious. We would have to start debugging
Do you mind getting a backtrace for the error? There are instructions here
https://mooseframework.inl.gov/application_development/debugging.html

@laagesen for awareness
